docs/DeveloperGuide.md

```diff
@@ -495,10 +495,79 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Adding Reminders
+
+### Deleting Reminders
+**Prerequisite**: Manually add several reminders as specified in the [Adding Reminders](#adding-reminders) section.
+1. Deleting a reminder that is present
+   1. Test case: `delete_reminder 1`<br>
+      Expected: First reminder is deleted from the reminder list. Details of the deleted contact will be displayed. Remaining reminders below the deleted reminder are shifted up by 1 position.
+   1. Alternative command to try: `dr`, `dr x`, `...` (where x is at between 1 and the size of the reminder list)<br>
+      Expected: similar to above
+2. Deleting a reminder that is not present
+   1. Test case: `delete_reminder 0`<br>
+      Expected: No reminder is deleted. Error status displayed.
+   1. Other incorrect delete_reminder commands to try: `delete_reminder`, `delete_reminder x`, `...`
+      (where x is larger than the size of the reminder list)<br>
+      Expected: No reminder is deleted. Error status displayed.
+   1. Alternative command to try: `dr`, `dr x`, `...` (where x is larger than the size of the reminder list)<br>
+      Expected: similar to above
+
+
 ### Saving data
 
 1. Dealing with missing/corrupted data files
 
    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
 1. _{ more test cases …​ }_
+
+### Saving sort preference
+1. **Saving valid sort preferences**
+   1. **Prerequisites**: Ensure multiple persons are listed using the `list` command.
```

Comment:
I think maybe can change the language a little cause it sounds like the user has to list the people then they can sort. Something like "Ensure there are  people in your address book, you can check so using the `list` command.

Reply:
Yep agreed.